src/app/models/route-location.ts

```diff
@@ -0,0 +1,4 @@
+export interface RouteLocation {
+  name: string;
+  place_id: string;
```

Comment:
I'd rather call it locationID then?

Reply:
to not cause confusion I would go with the api's naming schema, eg [place_id](https://developers.google.com/places/web-service/place-id)